src/core/ext/filters/client_channel/lb_policy/weighted_target/weighted_target.cc

```diff
@@ -559,6 +559,7 @@ void WeightedTargetLb::WeightedChild::DeactivateLocked() {
   // Start a timer to delete the child.
   Ref(DEBUG_LOCATION, "WeightedChild+timer").release();
   delayed_removal_timer_callback_pending_ = true;
+  ExecCtx::Get()->InvalidateNow();
```

Comment:
This is not a repeated timer, so there can't be a loop here.

Reply:
removed